cinema_service/settings.py

```diff
@@ -119,7 +119,7 @@
 
 USE_I18N = True
 
-USE_TZ = True
+USE_TZ = False
```

Comment:
The `USE_TZ` setting is set to `False`. It's recommended to set `USE_TZ = True` to enable timezone-aware datetimes, which helps prevent potential issues with time zone handling.